pr-data.csv

```diff
@@ -2686,7 +2686,7 @@ https://github.com/apache/incubator-kie-drools,8440dc2d4b95412a05a3c7b9501993e1a
 https://github.com/apache/incubator-kie-drools,8440dc2d4b95412a05a3c7b9501993e1ace9b496,drools-model/drools-model-codegen,org.drools.model.codegen.execmodel.GroupByTest.testCompositeKey,ID,,,
 https://github.com/apache/incubator-kie-drools,8440dc2d4b95412a05a3c7b9501993e1ace9b496,drools-model/drools-model-codegen,org.drools.model.codegen.execmodel.GroupByTest.testDecomposedGroupByKey,ID,,,
 https://github.com/apache/incubator-kie-drools,8440dc2d4b95412a05a3c7b9501993e1ace9b496,drools-model/drools-model-codegen,org.drools.model.codegen.execmodel.GroupByTest.testDecomposedGroupByKeyAnd2Accumulates,ID,,,
-https://github.com/apache/incubator-kie-drools,8440dc2d4b95412a05a3c7b9501993e1ace9b496,drools-model/drools-model-codegen,org.drools.model.codegen.execmodel.GroupByTest.testDecomposedGroupByKeyAnd2AccumulatesInConsequence,ID,,,
+https://github.com/apache/incubator-kie-drools,8440dc2d4b95412a05a3c7b9501993e1ace9b496,drools-model/drools-model-codegen,org.drools.model.codegen.execmodel.GroupByTest.testDecomposedGroupByKeyAnd2AccumulatesInConsequence,ID,Opened,https://github.com/apache/incubator-kie-drools/pull/6119,The fix is identical to one of the opened PR and I choose not to create a new PR to avoid duplication.
```

Comment:
What evidence (e.g., log files) do you have that the fix for another test also fixes this test?

Reply:
Will resolve in a separated PR